src/OpenLoco/Audio/Audio.cpp

```diff
@@ -914,28 +915,34 @@ namespace OpenLoco::Audio
 
             if (waterCount > kAmbientNumWaterTilesForOcean)
             {
-                ambientSound = PathId::css3;
+                newAmbientSound = PathId::css3;
             }
             else if (wildernessCount > kAmbientNumMountainTilesForWilderness)
             {
-                ambientSound = PathId::css2;
+                newAmbientSound = PathId::css2;
             }
             else if (treeCount > kAmbientNumTreeTilesForForest)
             {
-                ambientSound = PathId::css4;
+                newAmbientSound = PathId::css4;
             }
         }
         auto* channel = getChannel(ChannelId::ambient);
         if (channel == nullptr)
         {
             return;
         }
+
+        // TODO: Consider changing this so that we ask if the channel is playing a certain
```

Comment:
@ZehMatt / @LeftofZen  not sure what both of you were planning on doing with audio but it might be useful to be able to check if a channel is playing a certain buffer that would avoid the requirement for ambient noise to store in an extra global what source is playing.

Reply:
I was planning on adding some gui sliders in music options for the different channel volumes, but the channels are kinda all over the place atm so I am thinking of writing some kind of 'ChannelManager' class that bundles them together and ease of use. Could easily add a helper function in that class to do the buffer check you want

Reply:
No plans at the moment